egs/swbd/s5c/RESULTS

```diff
@@ -156,6 +156,14 @@ exit 0
 %WER 12.1 | 1831 21395 | 89.1 7.1 3.8 1.3 12.1 48.1 | exp/nnet3/tdnn_sp/decode_eval2000_hires_sw1_fsh_fg/score_12_0.0/eval2000_hires.ctm.swbd.filt.sys
 %WER 13.6 | 1831 21395 | 87.9 8.2 3.9 1.5 13.6 51.0 | exp/nnet3/tdnn_sp/decode_eval2000_hires_sw1_tg/score_11_0.0/eval2000_hires.ctm.swbd.filt.sys
 
+# results with nnet3 cnn+tdnn: local/nnet3/run_tdnn.sh (1.2.2016) (2 epoch training on speed-perturbed and volume perturbed data)
```

Comment:
with the basic recipe i got 
%WER 12.5 | 1831 21395 | 88.8 7.2 4.0 1.3 12.5 48.2 | exp/nnet3/tdnn_sp/decode_eval2000_hires_sw1_fsh_fg/score_13_0.0/eval2000_hires.ctm.swbd.filt.sys
%WER 23.7 | 2628 21594 | 79.0 14.7 6.3 2.7 23.7 59.5 | exp/nnet3/tdnn_sp/decode_eval2000_hires_sw1_fsh_fg/score_11_0.0/eval2000_hires.ctm.callhm.filt.sys
%WER 18.2 | 4459 42989 | 83.9 11.1 5.0 2.1 18.2 55.0 | exp/nnet3/tdnn_sp/decode_eval2000_hires_sw1_fsh_fg/score_11_0.0/eval2000_hires.ctm.filt.sys

%WER 13.8 | 1831 21395 | 87.7 8.3 4.0 1.5 13.8 50.9 | exp/nnet3/tdnn_sp/decode_eval2000_hires_sw1_tg/score_11_0.0/eval2000_hires.ctm.swbd.filt.sys
%WER 25.5 | 2628 21594 | 77.4 16.1 6.4 3.0 25.5 62.1 | exp/nnet3/tdnn_sp/decode_eval2000_hires_sw1_tg/score_10_0.0/eval2000_hires.ctm.callhm.filt.sys
%WER 19.7 | 4459 42989 | 82.5 12.1 5.4 2.2 19.7 57.3 | exp/nnet3/tdnn_sp/decode_eval2000_hires_sw1_tg/score_11_0.0/eval2000_hires.ctm.filt.sys
 
I need some time to run the experiment with matched updatable layers and parameters